### ISSUE TITLE
Syntax Error: Expected \":\", found String \":\"." when sending as string with latest @apollo/gateway package

### ISSUE THREAD

**Issue by kalpesh6641** [apollographql/federation#170]
**Description:**
We are seeing the following error with respect to the latest @apollo/gateway npm package version `0.20.1` when sending the query input as string for example.

**Query**
```

query {

   someEvents(input: "{\"name\":\"john\",\"age\":\"22\",\"class\":\"mca\"}")

}

```
**ERROR**
We see following error:

```

Syntax Error: Expected \":\", found String \":\"." when sending as string

```

But the same works fine with the @apollo/gateway gateway version `0.19.1`. Not sure if something has been updated with the latest version.

**Comment by Enrico2** [comment]
Hi @kalpesh6641, thanks so much for the bug report! Version `0.20.1` introduced our new query planner written in Rust, and you've found a real regression for which we didn't have a test case.
We'll get a release out asap with the fix!

**Comment by Enrico2** [comment]
@kalpesh6641 The fix should now be available in `0.20.2`

**Comment by kalpesh6641** [comment]
@Enrico2 thank you, this worked :)
